### DISCUSSION TITLE
FindMarkers for integrated biologic replicates

### DISCUSSION THREAD

**Discussion by mzz92** [satijalab/seurat#5106] [Q&A] [answered]
I have multiple 10X GEX datasets that are essentially biological replicates. These are all "control" data, and I am trying to find differentially expressed genes between UMAP clusters. Is it ok to integrate these datasets (say via SCT), then use FindMakers? If so, can "SCT" be used for this, or should "RNA" be used?

I've seen several responses on not using FindMarkers on integrated data for "control vs treatment" data. Just want to emphasize that this is not what I'd like to do; I'm looking at batched replicates.

My question was posed previously (https://github.com/satijalab/seurat/issues/340), but would like an updated viewpoint with v4.

Thanks.

**Comment by saketkc** [comment, marked as answer]
The [Integration vignette](https://satijalab.org/seurat/articles/integration_introduction.html) has a case study on stiumulated and control population which should cover your use case. Once integrated, you should switch to the RNA assay, run `NormalizeData` and then run `FindMarkers`.

**Reply by mzz92** [reply]
Thanks!

**Reply by myshu2017-03-14** [reply]
@saketkc hi, I can not find `NormalizeData` step in the "[Integration vignette](https://satijalab.org/seurat/articles/integration_introduction.html)". What should I do before run FindMarkers after switch to the RNA assay?

**Comment by ChristophH** [comment]
As an alternative approach, I'd suggest using the raw counts of the RNA assay and creating pseudobulk data. For motivation see [Squair et al., 2021](https://www.nature.com/articles/s41467-021-25960-2). For an implementation check out the [Libra package](https://github.com/neurorestore/Libra).
